### PR TITLE
chore: replace `anyhow` with `color_eyre`

### PR DESCRIPTION
**Pull request by zacharyftw**
Removed `anyhow` from places from the documentation part which i didnt notice in the 1st place while making the changes

### CODE REVIEW

File: src/content/docs/tutorials/counter-app/_multiple-functions.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ type Result<T> = std::result::Result<T, Err>;
 you can simply import `Result` from `anyhow`:
 
 ```rust
-use anyhow::Result;
+use color_eyre::eyre::Result;
```
Comment (joshka) [inline src/content/docs/tutorials/counter-app/_multiple-functions.md:50]: I'd generally shorten this to color_eyre::Result (it doesn't show up as a re-export in docs.rs as a doc hidden attribute on it, but it is recommended in various parts of the color_eyre docs oddly).

Not a huge deal, just one for consistency.

**Review by joshka — APPROVED** [review]

**Review by orhun — APPROVED** [review]
